kbase-extension/static/kbase/custom/custom.css

```diff
@@ -122,7 +122,7 @@ div.cell.selected.kb-error {
   border-left: 5px solid #d9534f;
 }
 
-#notebook {
+div#notebook {
```

Comment:
jupyter notebook css uses element+ID for selectors, EVEN THOUGH IDS ARE SUPPOSED TO BE UNIQUE! 🤦